EIPS/eip-6800.md

```diff
@@ -136,6 +133,22 @@ def old_style_address_to_address32(address: Address) -> Address32:
 
 #### Header values
 
+Account header values include:
+- `version`
+- `balance`
+- `nonce`
+- `code_keccak`
+- `code_size`
+
+The `version`, `balance`, `nonce` and `code_size` values are compressed in a single `basic_data` 32-byte value to be stored in the `BASIC_DATA_LEAF_KEY` sub-index.
+This value is encoded as:
+- `basic_data[0]`: `version` (8 bits)
+- `basic_data[1:4]`: are unset (i.e: zeroes).
+- `basic_data[4:8]`: `code_size` (32 bits)
+- `basic_data[8:16]`: `nonce` (64 bits)
+- `basic_data[16:32]`: `balance` (128 bits)
```

Comment:
It was also mentioned considering 96 bits -- just chose here the 128 bit version.